addons/binding/org.openhab.binding.mailserver/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="mailserver"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>MailServer Binding</name>
+    <description>This is the binding for MailServer.</description>
```

Comment:
I guess the idea was to maybe support these in a later version. I'd second the decision but yes, the `<description>` should be more precise.